Core/clim-core/panes/layout-protocol.lisp

```diff
@@ -435,24 +435,20 @@
 ;;;
 ;;; change-space-requirements (pane) :=
 ;;;   clear space requirements cache
-;;;   call change-space-requirements on parent pane
 ;;;   call note-space-requirements-changed
 ;;;
 ;;; This is split into :before, primary and :after method to allow for
 ;;; easy overriding of change-space-requirements without needing to
 ;;; know the details of the space requirement cache and the
 ;;; note-space-requirements-changed notifications.
 ;;;
-;;; The calls to change-space-requirements travel all the way up to
-;;; the top-level-sheet-pane which then invokes the layout protocol
-;;; calling layout-frame.
+;;; If :resize-frame t the calls to change-space-requirements travel
+;;; all the way up to the top-level-sheet-pane which then invokes the
+;;; layout protocol calling layout-frame.
 ;;;
```

Comment:
there is also a comment at the very top of the file which describes the layout protocol, maybe it should be amended too?

Reply:
The comment at the top is almost right, I only remove the sentence that note-space-requirements-changed is only a kind of signal.